ui/src/components/role/RoleSectionRow.js

```diff
@@ -187,6 +187,10 @@ class RoleSectionRow extends React.Component {
             this,
             `/domain/${this.props.domain}/role/${this.state.name}/policy`
         );
+        let clickTag = this.onClickFunction.bind(
```

Comment:
we should also move these to the constructor, I don't see any dynamic state value associated with these